esp-hal-smartled/Cargo.toml

```diff
@@ -35,3 +35,13 @@ esp32h2 = ["esp-hal/esp32h2"]
 esp32s2 = ["esp-hal/esp32s2"]
 ## Target the ESP32-S3.
 esp32s3 = ["esp-hal/esp32s3"]
+
+[patch.crates-io]
```

Comment:
Since we're using git dependencies and not patches in `esp-hal`, I don't think these should be required?

Reply:
Ah true! I always forget about that crate 👍